docs/it-manual/logs.md

```diff
@@ -0,0 +1,15 @@
+# How to check AWS logs (staging)
```

Comment:
**suggestion (blocking):** This heading mentions staging, but the doc page is in the "IT Manual" section where the intended audience is maintainers of downstream deployments. Suggest moving the doc to the "Developer Guide" section and also adding a link to it in the side nav

Reply:
Hi Bion, I made the changes that you suggested.